client/app/views/config_applications.coffee

```diff
@@ -54,6 +54,7 @@ module.exports = class ConfigApplicationsView extends BaseView
             @updateBtn.show()
         else
             @updateBtn.hide()
+            @updateBtn.show()
```

Comment:
@frankrousseau it looks weird to do `@updateBtn.show()` on both case (in the `if` and the `else`). Are you sure about that?